activity/settings.gradle

```diff
@@ -15,14 +15,23 @@
  */
 
 // see ../playground-common/README.md for details on how this works
+pluginManagement {
+    includeBuild '../playground-common/playground-plugin'
+}
```

Comment:
Please move to " quotes instead of ' for consistency. In all of our code we prefer ". Here and all the other files.

Reply:
Done